src/lib.rs

```diff
@@ -134,9 +137,13 @@ impl Order {
             .await?;
 
         self.nonce = nonce_from_response(&rsp);
-        let body = hyper::body::to_bytes(Problem::from_response(rsp).await?).await?;
+        let body_bytes = Problem::from_response(rsp)
```

Comment:
Let's keep this named as `body` at least in this commit to avoid unrelated changes.